db/migrate/20230817174049_ensure_javascript_cache_is_unique_per_theme.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class EnsureJavascriptCacheIsUniquePerTheme < ActiveRecord::Migration[7.0]
+  def change
+    remove_index :javascript_caches, :theme_id
+    add_index :javascript_caches, :theme_id, unique: true
```

Comment:
Is there any chance this could break an existing site because we have more than a cache for a theme?

Reply:
It's possible, so I ran a script to check that this hadn't happened